checks.d/fluentd.py

```diff
@@ -45,13 +47,13 @@ def check(self, instance):
             status = r.json()
 
             for p in status['plugins']:
-                tag = "%s:%s" % (tag_by, p.get(tag_by))
+                custom_tags.append("%s:%s" % (tag_by, p.get(tag_by)))
```

Comment:
Here, you'd append to custom_tags for each `p in status['plugins']`, probably what you want is to just send the original custom_tags + ["%s:%s" % (tag_by, p.get(tag_by))] for that `p in status['plugins']`. I believe we should leave `custom_tags` untouched.